tests/modules/graphql_file_validate/graphql_file_validate.module

```diff
@@ -8,9 +8,9 @@
 use Drupal\file\FileInterface;
 
 /**
- * Implements hook_file_validate().
+ * Test to validate file exists.
  */
-function graphql_file_validate_file_validate(FileInterface $file): void {
+function graphql_file_validate_test_file(FileInterface $file): void {
```

Comment:
please remove the module file and move this code directly into the even subscriber.